numpy/random/mtrand.pyx

```diff
@@ -643,7 +642,7 @@ cdef class RandomState:
 
     def randint(self, low, high=None, size=None, dtype=int):
         """
-        randint(low, high=None, size=None, dtype='l')
+        randint(low, high=None, size=None, dtype=np.int_)
```

Comment:
Apologies for the incorrect advice earlier - this should have been `int` after all, as should 673. These repeated signature lines are confusing.

Reply:
Thanks :D